content/docs/command-reference/remove.md

```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        `.dvc` files to remove.
 ```
 
 ## Description
 
 This command safely removes data files or directories that are tracked by DVC
 from the <abbr>workspace</abbr>. It takes a
 [DVC-File](/doc/user-guide/dvc-file-format) as input, removes all of its outputs
-(`outs`), and optionally removes the DVC-file itself.
+(`outs`), and optionally removes the `.dvc` file itself.
```

Comment:
> Moving from https://github.com/iterative/dvc.org/pull/1403#discussion_r437021912:

Also, per https://github.com/iterative/dvc.org/pull/1403#pullrequestreview-426045191 this whole sentence is wrong actually. It should be:

> It takes a stage name (see `-n` option of `dvc run`) or a [`.dvc` file](/doc/user-guide/dvc-file-format) as `target`, removes all of its outputs (`outs` field), and optionally removes the stage entry in [`dvc.yaml`(/doc/user-guide/dvc-file-format) or the `.dvc` file itself.

Reply:
Wait. Per https://github.com/iterative/dvc.org/pull/1403#discussion_r437033424 that was also not correct... It should be:

> It takes a stage name (see -n option of dvc run) or a .dvc file as target, removes the stage entry in [dvc.yaml(/doc/user-guide/dvc-file-format) or the .dvc file itself and optionally removes the outputs.

(all appropriate links though)

Reply:
Incomplete. Left another comment.